dockerfiles/default-configs/node.yaml

```diff
@@ -0,0 +1,72 @@
+# [time] is a concatenation of spans. Where each span is an integer number and a suffix.
+# E.g.: [1y 2M 4w 8d 16h 32min 64s 128ms 256us]
+# Supported suffixes:
+#  nsec, ns -- microseconds
+#  usec, us -- microseconds
+#  msec, ms -- milliseconds
+#  seconds, second, sec, s
+#  minutes, minute, min, m
+#  hours, hour, hr, h
+#  days, day, d
+#  weeks, week, w
+#  months, month, M -- defined as 30.44 days
+#  years, year, y -- defined as 365.25 days
+
+# [file] logger config file
+log_config: /configs/logger.yaml
```

Comment:
I suggest to use `/bob/configs/` directory. This way all files associated with bob will be in `/bob/` directory